cmd/start.go

```diff
@@ -63,6 +72,11 @@ func Start(repoName string, tp node.Type) *cobra.Command {
 	}
 
 	cmd.Flags().String(cfgAddress, "", "Indicates node to connect to the given remote core node")
+	cmd.Flags().String(
+		loglevel,
+		"info",
+		"DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL, and\n// their lower-case forms.",
```

Comment:
I'm assuming the `\n//` just creates a new-line?

Reply:
Yeah